sysinit/usage.ml

```diff
@@ -10,8 +10,9 @@
 
 let version () =
   Printf.printf "The Coq Proof Assistant, version %s (%s)\n"
-    Coq_config.version Coq_config.date;
-  Printf.printf "compiled on %s with OCaml %s\n" Coq_config.compile_date Coq_config.caml_version
+    Coq_config.version (Option.default "n/d" Coq_config.date);
+  Printf.printf "compiled on %s with OCaml %s\n"
+    (Option.default "n/d" Coq_config.compile_date) Coq_config.caml_version
```

Comment:
Could we print nothing instead of `on n/d`?

Reply:
I was wondering about this, as what could case less breakage to tools possibly parsing this etc...

I think I do prefer to print something, for example the default for many users of `dune-build-info` is to print
```
version: n/a
```
when the version is not available, so I dunno, I still have a small preference for printing "n/a"

Reply:
n/d or n/a?

Reply:
I've changed it to `n/a` which is more correct.